loops-arrays-objects.js

```diff
@@ -31,9 +32,21 @@ const scores = [
 // A
 // D
 // ==========================================
-
-
-
+for (let i = 0; i < scores.length; i++) {
+    if (scores[i].score < 60) {
+        console.log('F');
+        } else if (scores[i].score >= 60 && scores[i].score < 70) {
+            console.log("D");
+        } else if (scores[i].score >= 70 && scores[i].score < 80) {
+            console.log("C");
+        } else if (scores[i].score >= 80 && scores[i].score < 90) {
+            console.log("B");
+        } else if (scores[i].score >= 90 && scores[i].score <= 100) {
+            console.log("A");
+        } else {
+            console.error('er gaat iets mis')
+        }
+    }
 
```

Comment:
In de condities van je if-statements voer je een dubbele (overbodige) check uit. Wat rechts van de && operator staat is voldoende.  Zie uitwerkingen: 

```for (let i = 0; i < scores.length; i++) {
    if (scores[i].score < 60) {
        console.log('F');
    } else if (scores[i].score < 70) {
        console.log('D');
    } else if (scores[i].score < 80) {
        console.log('C');
    } else if (scores[i].score < 90) {
        console.log('B');
    } else {
        console.log('A');
    }
}